iceoryx_hoofs/test/moduletests/test_cxx_variant.cpp

```diff
@@ -428,6 +442,8 @@ TEST_F(variant_Test, CreatingSecondObjectViaMoveCTorResultsInTwoDTorCalls)
         {
             iox::cxx::variant<int, DTorTest> schlomo(std::move(ignatz));
             EXPECT_THAT(DTorTest::dtorWasCalled, Eq(false));
+            // check if move is invalidating the object
+            // NOLINTNEXTLINE (bugprone-use-after-move)
             EXPECT_THAT(ignatz.index(), Eq(1U));
```

Comment:
@elfenpiff Just checking here, to make sure that my understanding is correct: The moved from object `ignatz` will not be invalidated but we rely on the move c'tor from the stored type to properly invalidate the moved from object. This means the index will not be touched, ack?